rpmconf/Install/Util/utilfunc.sh

```diff
@@ -2383,7 +2383,10 @@ getInstallPackages() {
       if [ $? = 0 ]; then
         echo "    Upgrading $i"
         if [ $i = "zimbra-network-modules-ng" ]; then
-          INSTALL_PACKAGES="$INSTALL_PACKAGES zimbra-talk"
+            askInstallPkgYN "Install zimbra-talk" "yes" "Y" "N"
```

Comment:
Need to make sure that can't install both zimbra-talk and zimbra-chat.  I notice other places where `getChatOrTalkPackage` is called.  Some of the formatting is bad around those calls (not this pull request) could do with changing tabs to spaces and fixing.  `getChatOrTalkPackage` could also do with a comment that it expects response to be setup (yes if want zimbra-network-modules-ng installed)

Reply:
Yes, I have made sure that both of them will not get installed only either one of them based on what has been selected. And if chat is already installed then before installing talk it will remove the chat first. It has been also passed in System-Integration testing.

I will fix the formatting.